src/controllers/movieController.ts

```diff
@@ -0,0 +1,160 @@
+import { Request, Response } from 'express';
+import { validationResult } from 'express-validator';
+import config from '../config';
+import logger from '../utils/logger';
+import HttpService from '../services/httpService';
+import StorageService, { StorageError } from '../services/storageService';
+import { CreateRequestBody, GetAllRequestQuery, SortOrder } from './types';
+import { FakeMovieListType } from '../data/fakeMovieList';
+import { customComparer } from './helpers';
+
+class MovieController {
+  get = async (req: Request, res: Response) => {
+    try {
+      const validationErrors = this.applyValidationResult(req, res);
```

Comment:
better to create validation middleware for that

Reply:
Thanks for advice!